### PR TITLE
Added mappings for VillagerEntity inner class

### PR DESCRIPTION
**Pull request by freundTech**

### CODE REVIEW

File: mappings/net/minecraft/entity/passive/VillagerEntity.mapping [hunk 1]
```diff
@@ -1,4 +1,13 @@
 CLASS aui net/minecraft/entity/passive/VillagerEntity
+	CLASS aui$a GolemSpawnContition
```
Comment (Runemoro) [inline mappings/net/minecraft/entity/passive/VillagerEntity.mapping]: Condition

File: mappings/net/minecraft/entity/passive/VillagerEntity.mapping [hunk 2]
```diff
@@ -1,4 +1,13 @@
...
+		FIELD a lastWorked J
+		FIELD b lastSlept J
+		METHOD a setLastWorked (J)V
+			ARG 1 lastWorked
+		METHOD b setLastSlept (J)V
+			ARG 1 lastSlept
+		METHOD c checkEligibleForSpawn (J)Z
```
Comment (Runemoro) [inline mappings/net/minecraft/entity/passive/VillagerEntity.mapping]: isEligibleForSpawn?

Comment (Runemoro) [reply mappings/net/minecraft/entity/passive/VillagerEntity.mapping]: Or why not just canSpawn?

File: mappings/net/minecraft/entity/passive/VillagerEntity.mapping (hunk 1 above)
Comment (freundTech) [reply mappings/net/minecraft/entity/passive/VillagerEntity.mapping]: Fixed

File: mappings/net/minecraft/entity/passive/VillagerEntity.mapping (hunk 2 above)
Comment (freundTech) [reply mappings/net/minecraft/entity/passive/VillagerEntity.mapping]: Changed it to canSpawn

**Review by Gegy — APPROVED** [review]

**Pull request merged**